assets/css/general/basics.css

```diff
@@ -16,10 +16,18 @@
     --twitter-color: #1da1f2;
     --rss-color: #f26522;
     --animation-base: ease-in-out;
-    --font-base: Mulish, -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif;
+    --font-base: Mulish, -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica,
+        Arial, sans-serif;
```

Comment:
No reformatting, please. I'd like to keep in sync with upstream template as much as possible.